packages/multichain-account-service/src/MultichainAccountService.ts

```diff
@@ -293,6 +302,57 @@ export class MultichainAccountService {
     return Array.from(this.#wallets.values());
   }
 
+  /**
+   * Creates a new multichain account wallet with the given mnemonic.
+   *
+   * NOTE: This method should only be called in client code where a mutex lock is acquired.
+   * `discoverAndCreateAccounts` should be called after this method to discover and create accounts.
+   *
+   * @param options - Options.
+   * @param options.mnemonic - The mnemonic to use to create the new wallet.
+   * @throws If the mnemonic has already been imported.
+   * @returns The a tuple of the new multichain account wallet and the entropy source id.
+   */
+  async createMultichainAccountWallet({
+    mnemonic,
+  }: {
+    mnemonic: string;
+  }): Promise<
+    [MultichainAccountWallet<Bip44Account<KeyringAccount>>, EntropySourceId]
```

Comment:
I think returning the wallet is enough, since we can get the entropy source from it afterward (with `wallet.entropySource`).